cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/extensions/CimCharacteristicsMergingView.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.extensions;
+
+import com.google.auto.service.AutoService;
+import com.powsybl.cgmes.conversion.elements.CgmesTopologyKind;
+import com.powsybl.iidm.mergingview.extensions.ExtensionMergingView;
+import com.powsybl.iidm.network.Network;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+@AutoService(ExtensionMergingView.class)
+public class CimCharacteristicsMergingView implements ExtensionMergingView<Network, CimCharacteristics> {
+
+    @Override
+    public Class<? extends CimCharacteristics> getExtensionClass() {
+        return CimCharacteristics.class;
+    }
+
+    @Override
+    public String getName() {
+        return "cimCharacteristics";
+    }
+
+    @Override
+    public void merge(Network extendable, CimCharacteristics extension) {
+        CimCharacteristics original = extendable.getExtension(CimCharacteristics.class);
+        if (original == null) {
+            extendable.newExtension(CimCharacteristicsAdder.class)
+                    .setCimVersion(extension.getCimVersion())
+                    .setTopologyKind(extension.getTopologyKind())
+                    .add();
+        } else {
+            extendable.newExtension(CimCharacteristicsAdder.class)
+                    .setCimVersion(Math.min(original.getCimVersion(), extension.getCimVersion()))
```

Comment:
I think that the CIM version should be identical. How does it work if not?

Reply:
14 is less extensive than 16, so if one of the file is 14, I make the merging view 14.

Reply:
But do we really want to support that ? Have we perform some tests with this kind of hybrid association ?

Reply:
Good question, maybe not. Do you think it should throw an exception in this case?

Reply:
Yes, it is better.